labonneboite/web/contact_form/views.py

```diff
@@ -46,9 +44,8 @@ def change_info_or_apply_for_job(siret=None):
 @contactFormBlueprint.route('/connexion-recruteur/<siret>', methods=['GET'])
 def ask_recruiter_pe_connect(siret=None):
     """
-    Ask user if he wants to change company information or apply for a job,
-    in order to avoid the change_info page to be spammed so much by
-    people thinking they are actually applying for a job.
+    Ask recruiter if he/she wants to be connected with his Pole Emploi recruiter account
+    in order to validate his/her identity more quickly/easely
```

Comment:
pin.: pour éviter le he/she on peut utiliser "they". Ca donnerait : "Ask recruiters if they want to be connected with their Pole Emploi recruiter account in order to validate their identity more quickly/easily"